pkg/schema/schema_test.go

```diff
@@ -1,4 +1,108 @@
 package schema
 
-// TODO tests for this stuff! Everything in this package is totally generic,
-// nothing is specific to Grafana
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"cuelang.org/go/cue"
+	"github.com/google/go-cmp/cmp"
+	"golang.org/x/tools/txtar"
+)
+
+var CasesDir = filepath.Join("load", "testdata", "artifacts", "dashboards", "trimdefault")
+
+type Case struct {
+	Name   string
+	CUE    string
+	Full   string
+	Trimed string
```

Comment:
nit: spelling. (PR needs this in a few places)
```suggestion
	Trimmed string
```